packages/core/src/components/logo/readme.md

```diff
@@ -0,0 +1,49 @@
+# dso-logo
+
+Een Web Component waarmee een slim, toegankelijk en uniform logo in applicaties kan worden getoond.
+
+## Gebruik
+
+Enkel het component zelf gebruiken is voldoende. Dit toont alleen het beeldmerk (de 'target') en het woordmerk (de tekst "Omgevingsloket" in de kleur grasgroen, waarvan het woord "loket" in bosgroen):
+
+```html
+<dso-logo></dso-logo>
+```
+
+Eventueel kunnen een `label` (applicatienaam, portaalnaam of kruimelpadlocatie) en `ribbon` (status of servernaam) worden meegegeven:
+
+```html
+<dso-logo label="Regels op de kaart" ribbon="Beta"></dso-logo>
+```
+
+Een `label` toont de titel of naam van de applicatie waar je je als bezoeker in bevindt.
+
+Een `ribbon` is een soort sticker die overlapt met het woordmerk, waar een maximaal vijf-letterige tekst in kan komen te staan om de status van de pagina aan te duiden. Bijvoorbeeld "Beta", "Test", "Acc", of "Pre".
```

Comment:
Waar komt de restrictie van 5 letterige tekst vandaan? In de code doen we er niets mee.

In de documentatie zie ik zowel "beta" als "Beta" voorbijkomen, ik denk dat we er ééntje moeten kiezen.

Reply:
Het lijkt me goed om in ieder geval een suggestie te geven voor een maximum lengte, net als dat we aangeven wat het idee is van de ribbon. Ik heb er van gemaakt "Suggested maximum length: 5 characters."